src/components/calcite-pick-list-item/calcite-pick-list-item.scss

```diff
@@ -1,74 +1,81 @@
 :host {
-  @extend %component-host;
-  @apply items-stretch
+	@extend %component-host;
+	@apply items-stretch
     flex
     text-color-1
     shadow-border-bottom
     m-0
     mb-px;
-  transition: background-color $transition;
-  animation: calcite-fade-in $transition;
+	transition: background-color $transition;
```

Comment:
Are we able to use values from the Tailwind config here?

Reply:
For the transition?

Reply:
Handling in a separate PR.